index.html

```diff
@@ -324,25 +324,25 @@ <h2 class="experience-sub-title project-title">Project Two</h2>
         <div class="details-container color-container">
           <div class="article-container">
             <img
-              src="./assets/project-3.png"
+              src="./assets/troubadour.png"
               alt="Project 3"
               class="project-img"
             />
           </div>
-          <h2 class="experience-sub-title project-title">Project Three</h2>
+          <h2 class="experience-sub-title project-title">Troubadour</h2>
           <div class="btn-container">
             <button
               class="btn btn-color-2 project-btn"
               onclick="location.href='https://github.com/'"
             >
               Github
             </button>
-            <button
+            <!-- <button
```

Comment:
It's cleanest to remove any commented out code. Comments should generally be reserved for explaining things that are not inherently obvious in your code. More reading [here](https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/) if you're interested!